.travis.yml

```diff
@@ -11,11 +11,11 @@ branches:
   - travis-test
 
 install:
-  - pip install -r requirements.txt
-  - make generate-proto
+  - python -m pip install --upgrade pip
```

Comment:
Is it essential?

Reply:
Sorry for temporary commit.

I have encountered a problem related to travis-ci.
I have no idea why unit tests are failed on travis whereas succeed on my local.

I'm trying to figure out problem by using the temporary commit.

Reply:
Added. 
Please check this commit and resolve if it is acceptable.

https://github.com/icon-project/loopchain/pull/476/commits/540346d7ad68e846819c2dd14d57930158f7ce5d